### ISSUE TITLE
Hijacking of built in mappings

### ISSUE THREAD

**Issue by nl253** [vimwiki/vimwiki#341]
Hi, 

I just wanna say, this is a great plugin, but I am used to certain vim features like moving up and down the jump list with `<C-i>` or `<Tab>` or aligning with `=`. The issue is, usual autocommands don't seem to remap it:

```vim
function! VimWikiInit()
    nnoremap <C-i> <C-i>
    nnoremap <Tab> <Tab>
    nnoremap <S-Tab> <S-Tab>
    nnoremap = =
endfunction

    au FileType vimwiki,vimwiki_markdown,vimwiki_markdown_custom call VimWikiInit()
 ```

Its not just autocommands I tried manually typing it in ex mode and sill it wouldn't override 

```vim
@<Plug>VimwikiNextLink
```

is it possible at all to override vimwiki mappings ? 

I'd appreciate any tips,
Thanks

**Comment by EinfachToll** [comment]
Vimwiki does not map keys to a command when there is already a key for this command. So, putting this into .vim/ftplugin/vimwiki.vim should do the trick (or use your solution with the FileType autocommand).

```viml
nmap <F13> <Plug>VimwikiNextLink
nmap <F14> <Plug>VimwikiPrevLink
nmap <F15> <Plug>VimwikiAddHeaderLevel
```
It maps (supposedly) nonexisting keys to the commands. `<C-i>` is not used by Vimwiki.

**Comment by nl253** [comment]
That solved it, thanks. 

Once again, great plugin.

**Comment by Flurrywinde** [comment]
This doesn't work for me. I tried putting my own mapping to <Plug>VimwikiFollowLink in both .vimrc and .vim/ftplugin/vimwiki, but in both cases, <CR> still followed links. Could this be because I'm using Pathogen to load the vimwiki plugin? I tried putting the mapping to <Plug>VimwikiFollowLink before the Pathogen call in .vimrc, but it still didn't work.

I'm using the dev branch currently behind by 30 commits. (I don't know how better to tell you what version I'm using. I'm afraid to git pull since I worry these commits might introduce instability.)

I do see the if !hasmapto commands in the plugin code.

**Comment by EinfachToll** [comment]
The path to the ftplugin file is `.vim/ftplugin/vimwiki.vim`, you are missing the `.vim` file extension. (Also, the path is for Linux users, on Windows it's usually `vimfiles/ftplugin/vimwiki.vim` in your user directory). I use pathogen, too, and for me, it works. If you're unsure if the file is loaded at all, you can put a line like `echom "hello"` in this file. Now, if you open a vimwiki buffer with Vim, you should see this message.

**Comment by EinfachToll** [comment]
The path to the ftplugin file is `.vim/ftplugin/vimwiki.vim`, you are missing the `.vim` file extension. (Also, the path is for Linux users, on Windows it's usually `vimfiles/ftplugin/vimwiki.vim` in your user directory). I use pathogen, too, and for me, it works. If you're unsure if the file is loaded at all, you can put a line like `echom "hello"` in this file. Now, if you open a vimwiki buffer with Vim, you should see this message.

**Comment by Flurrywinde** [comment]
Oops, the missing `.vim` extension was a typo, but I had it on the actual file. I added the `echom "hello"`, and it did show up when I open a `.wiki` file. `<CR>` still calls VimwikiFollowLink despite having `nmap <space>fl <Plug>VimwikiFollowLink` in it. 

I then did a git pull to get the current dev branch, and it still happens. (I exited and ran vim again after each of these changes.)

Next, I removed all other plugins, and voila, it worked! I then narrowed the culprit down to vim-stay. 

I posted an issue on the vim-stay github page, but I'm wondering if you might know why this might be happening too? I like both plugins a lot and would really like to fix this.
